addons/admissions-webhook/patch.json

```diff
@@ -0,0 +1 @@
+{"admissionConfig":{"pluginConfig":{"ValidatingAdmissionWebhook":{"configuration":{"apiVersion":"apiserver.config.k8s.io/v1alpha1","kind":"WebhookAdmission","kubeConfigFile":"/dev/null"}},"MutatingAdmissionWebhook":{"configuration":{"apiVersion":"apiserver.config.k8s.io/v1alpha1","kind":"WebhookAdmission","kubeConfigFile": "/dev/null"}}}}}
```

Comment:
@anjannath not in a single line but should be as pretty print, so it would be easy to modify or add in future.

```json
$ echo '{"admissionConfig":{"pluginConfig":{"ValidatingAdmissionWebhook":{"configuration":{"apiVersion":"apiserver.config.k8s.io/v1alpha1","kind":"WebhookAdmission","kubeConfigFile":"/dev/null"}},"MutatingAdmissionWebhook":{"configuration":{"apiVersion":"apiserver.config.k8s.io/v1alpha1","kind":"WebhookAdmission","kubeConfigFile": "/dev/null"}}}}}' | python -mjson.tool
{
    "admissionConfig": {
        "pluginConfig": {
            "MutatingAdmissionWebhook": {
                "configuration": {
                    "apiVersion": "apiserver.config.k8s.io/v1alpha1",
                    "kind": "WebhookAdmission",
                    "kubeConfigFile": "/dev/null"
                }
            },
            "ValidatingAdmissionWebhook": {
                "configuration": {
                    "apiVersion": "apiserver.config.k8s.io/v1alpha1",
                    "kind": "WebhookAdmission",
                    "kubeConfigFile": "/dev/null"
                }
            }
        }
    }
}
```

Reply:
Okay.. will change it, i was afraid the newlines are gonna mess with the patch, let me try that now!

Reply:
Done!